docs/csharp/programming-guide/strings/snippets/VerbatimStringInterpolation.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace StringExamples;
+public static class VerbatimInterpolation
+{
+    public static void VerbatimInterpolationExample()
+    {
+        // <VerbatimStringInterpolation>
+        var jh = (firstName: "Jupiter", lastName: "Hammon", born: 1711, published: 1761);
+        Console.WriteLine($@"{jh.firstName} {jh.lastName}
+            was an African American poet born in {jh.born}.");
+        Console.WriteLine(@$"He was first published in {jh.published}
+        at the age of {jh.published - jh.born}.");
```

Comment:
Another alternative is to outdent the 2nd lines:

```suggestion
        Console.WriteLine($@"{jh.firstName} {jh.lastName}
was an African American poet born in {jh.born}.");
        Console.WriteLine(@$"He was first published in {jh.published}
at the age of {jh.published - jh.born}.");
```